js/ui/grid_core/ui.grid_core.columns_view.js

```diff
@@ -918,15 +923,15 @@ const columnsViewMembers = {
         return this.option('templatesRenderAsynchronously') && this.option('renderAsync') === false;
     },
 
-    waitAsyncTemplates: function(change, forceWaiting) {
+    waitAsyncTemplates: function(forceWaiting) {
```

Comment:
If it's possible I suggest rewriting it to the more clear version
waitAsyncTemplates: function(forceWaiting = false) {